packages/paste-core/components/skeleton-loader/src/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useSpring, animated} from '@twilio-paste/animation-library';
+import {css, styled} from '@twilio-paste/styling-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {LayoutProps, MarginProps, BorderRadiusProps} from '@twilio-paste/style-props';
+
+const AnimatedSkeleton = animated(Box);
+
+const StyledAnimatedSkeleton = styled(AnimatedSkeleton)(() =>
+  css({
+    background: `linear-gradient(90deg, transparent, rgba(255, 255, 255, 0.3) 40%, rgba(255, 255, 255, 0.3) 60%, transparent)`,
+  })
+);
+
+export interface SkeletonLoaderProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    LayoutProps,
+    MarginProps,
```

Comment:
I wonder if this is a good thing or a bad thing?

Should we make you use a layout like stack to set the gap between skeleton boxes?

I get that the reason we do individual is likely so that you can replicate the spacing between a heading and a paragraph to a box, which might be specific to the component you are mimicking.

But if we ever change those, your skeleton spacing in your product is going to go out of sync and look different to the component spacing anyway (unless you are willing to constantly go around nudging all your skeletons every time we adjust spacing). So why not just simplify this component? 🤔

Reply:
> I get that the reason we do individual is likely so that you can replicate the spacing between a heading and a paragraph to a box, which might be specific to the component you are mimicking.

That's the only reason I was using the margins on this one, but that could also be done by wrapping a Box around it. But that would also lead to the same issue if the spacing changes, right? Spacing would be different so the skeletons would be off.

That said, I do think I like the simpler is better approach and just use a Box if you need some extra spacing.